ideas.txt

```diff
@@ -29,6 +29,3 @@ https://github.com/maebert/jrnl
 
 data manipulation:
 https://github.com/sindresorhus/rgb-hex-cli
-
```

Comment:
@capaldo again, you commited from the wrong base branch :)

Reply:
I made the changes to both branches?

Reply:
You were still on this branch, while you made the change.
Did you mean to commit to master?

Reply:
No, I just removed the additions to both branches so on merge it wouldn't re-add them.